README.md

```diff
@@ -989,6 +989,83 @@ when the content is present.
   end
 ```
 
+## Custom Components
+
+When you use custom wrappers, you might also be looking for a way to add custom components to your 
+wrapper. The default components are:
+
+```
+:label         # The <label> tag alone
+:input         # The <input> tag alone
+:label_input   # The <label> and the <input> tags
+:hint          # The hint for the input
+:error         # The error for the input
+```
+
+A custom component might be interesting for you if your views look something like this:
+
+```ruby
+<%= simple_form_for @blog do |f| %>
+  <div class="row">
+    <div class="span1 number">
+      1
+    </div>
+    <div class="span8">
+      <%= f.input :title %>
+    </div>
+  </div>
+  <div class="row">
+    <div class="span1 number">
+      2
+    </div>
+    <div class="span8">
+      <%= f.input :body, :as => :text %>
+    </div>
+  </div>
+<% end %>
+```
+
+A cleaner method to create your views would be
+
+```
+<%= simple_form_for @blog, :wrapper => :with_numbers do |f| %>
+  <%= f.input :title, :number => 1 %>
+  <%= f.input :body, :as => :text, :number => 2 %>
+<% end %>
+```
+
+To use the number option on the input you have to create a new component.
+
+```
+module Numbers
+  # To avoid deprecation warning, you need to make the wrapper_options explicit
+  # even when they won't be used.
+  def number(wrapper_options = nil)
+    @number ||= begin
+      options[:number].to_s.html_safe if options[:number].present?
+    end
+  end
+end
+
+SimpleForm.include_component(Numbers)
+```
+
+Add a new wrapper to the config/initializers/simple_form.rb file.
+
+```
+config.wrappers :with_numbers, :tag => 'div', :class => 'row', :error_class => 'error' do |b|
```

Comment:
Just a nitpicking, but since this method seems to only configure one wrapper at a time, wdyt of `config.wrapper`? 🤔

Reply:
They are different APIs. `config.wrappers` is to define a new wrapper already `config.wrapper` is to retrieve some wrapper.

Reply:
Oh, right, you weren't creating the method here, you're just taking use of it 👍